app/assets/javascripts/views/story_view.js

```diff
@@ -579,6 +480,174 @@ module.exports = FormView.extend({
+      this.makeFormControl({
+        name: "labels",
+        label: true,
+        control: this.textField("labels"),
+        class: 'form-control',
+        disabled: this.isReadonly()
+      })
+    );
+
+    this.$el.append(
+      this.makeFormControl(this.makeDescription())
+    );
+
+    this.renderTasks();
+
+    this.$el.append(
+      this.makeFormControl(function(div) {
+        var random = (Math.floor(Math.random() * 10000) + 1);
+        var progress_element_id = "documents_progress_" + random;
+        var finished_element_id = "documents_finished_" + random;
+        var attachinary_container_id = "attachinary_container_" + random;
+
+        $(div).append(this.label('attachments', I18n.t('story.attachments')));
+        $(div).addClass('uploads');
+        if(!this.isReadonly()) {
+          $(div).append(this.fileField("documents", progress_element_id, finished_element_id, attachinary_container_id));
+          $(div).append("<div id='" + progress_element_id + "' class='attachinary_progress_bar'></div>");
+        }
+        $(div).append('<div id="' + attachinary_container_id + '"></div>');
+
+        // FIXME: refactor to a separated AttachmentView or similar
+        // must run the plugin after the element is available in the DOM, not before, hence, the setTimeout
+        clearTimeout(window.executeAttachinaryTimeout);
+        window.executeAttachinaryTimeout = setTimeout(executeAttachinary, 500);
+      })
+    );
+
+    this.initTags();
+
+    this.renderNotes();
+  },
+
+  makeStoryTypeSelect: function(div) {
+    var story_type_options = [];
+    _.each(["feature", "chore", "bug", "release"], function(option) {
+      story_type_options.push([I18n.t('story.type.' + option), option])
+    });
+    $(div).append(this.makeFormControl({
+      name: "story_type",
+      label: true,
+      disabled: true,
+      control: this.select("story_type", story_type_options, {
+        attrs: {
+          class: ['story_type'],
+          disabled: this.isReadonly()
+        }
+      })
+    }));
+  },
+
+  makeDescription: function()
+  {
+    return function(div) {
+      $(div).append(this.label("description", I18n.t('activerecord.attributes.story.description')));
+
+      if(this.model.isNew() || this.model.get('editingDescription')) {
+        var textarea = this.textArea("description");
+        $(textarea).atwho({
+          at: "@",
+          data: window.projectView.usernames(),
+        });
+        $(div).append(textarea);
+      }
+      else {
+        var $description = $('<div class="description-wrapper"><div>');
+        $(div).append($description);
+      }
+    }
+  },
+
+  renderReleaseStory: function() {
+
+    if(this.model.get('editing')) {
+      this.$el.append(
+        this.makeFormControl(function(div) {
+          this.makeStoryTypeSelect(div);
+        }));
+    }
+
+    const $storyDate = $('<div class="form-group" data-story-datepicker></div>');
+    this.$el.append(this.$el.append($storyDate));
```

Comment:
Is this really supposed to be appending twice? `this.$el.append(this.$el.append(...)`